### ISSUE TITLE
Input Touch Issue

### ISSUE THREAD

**Issue by stanleyjoy** [fehaar/FFWD#69]
I have used these codes and tested on my Lumia 610 and Lumia 900. I found a really strange bug. The count value that I am getting is not consistent. When i place a finger on the device the count value can go upto 4 during certain intervals.
Do you have any clue as to why this is happening???

XNA Code:
<code>
protected override void Update(GameTime gameTime)
        {
        TouchCollection touches = TouchPanel.GetState();
        System.Diagnostics.Debug.WriteLine("COUNT: "+touches.Count);
    }
</code>

Unity Code:
<code>
public override void Update () 
    {
        int count = Input.touchCount;
                Debug.Log("COUNT: "+count);
        }
</code>

Stan

**Comment by fehaar** [comment]
I reckon that the count from Game1 is consistantly 1 while the other one goes mad? That is a very good repro case to find the bug.

It might be some of the code that cleans up after the touches in Input.Update that has issues. I know this has been working at some point. Maybe some of the later updates has broken this code. There has been some XBox controller changes that were merged in not made by me.

**Comment by stanleyjoy** [comment]
I have used these in separate instances I found this issue when it was automatically calling functions like "tap to continue". At that time only Unity had the touch input. 
I just checked it on XNA with the Unity Script disabled and I still got the same result.
To fry your noodle even more, it was even happening on a New Windows Phone 7.1 game project.
Looks like the only hope is gestures.
Stan
P.S. Gestures still have issues.
